.github/workflows/ci-base-tests.yml

```diff
@@ -54,4 +54,59 @@ jobs:
             --ignore=./smarts/core/tests/test_smarts_memory_growth.py \
             --ignore=./smarts/env/tests/test_benchmark.py \
             --ignore=./smarts/env/tests/test_learning.py \
-            -k 'not test_long_determinism'
\ No newline at end of file
+            -k 'not test_long_determinism'
+
+  mac:
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+    strategy:
+      matrix:
+        tests:
+          - ./envision
+          - ./smarts/contrib
+          - ./tests
+          - ./smarts/sstudio --ignore=./smarts/sstudio/genhistories.py
+          - ./smarts/env/tests/test_rllib_hiway_env.py
+          - ./smarts/core
+          - ./smarts/env --ignore=./smarts/env/tests/test_rllib_hiway_env.py
+    runs-on: macos-11
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Python
+        run: |
+          brew update
+          brew install python@3.7
+          brew unlink python@3.9
+          brew link --force --overwrite python@3.7
+      - name: Setup SUMO
+        run: |
+          brew install xquartz
+          brew tap dlr-ts/sumo
+          brew install sumo spatialindex
+          brew install geos
+      - name: Install dependencies
+        run: |
+          python3.7 -m venv ${{env.venv_dir}}
+          . ${{env.venv_dir}}/bin/activate
+          pip install --upgrade pip
+          pip install --upgrade wheel
+          pip install -r utils/setup/mac_requirements.txt
+          pip install -e .[train,test,camera-obs]
+      - name: Run smoke tests
+        run: |
+          . ${{env.venv_dir}}/bin/activate
+          export SUMO_HOME="/usr/local/opt/sumo/share/sumo"
+          open -g -a XQuartz.app
+          make build-all-scenarios
+          PYTHONHASHSEED=42 pytest -v \
+            --doctest-modules \
+            -n auto \
```

Comment:
What lead to the removal of these options in mac test?
```
--forked \
--dist=no \
```

Reply:
Some of the core tests pass for Macos CI.